resolver/resolver.cc

```diff
@@ -1144,25 +1144,46 @@ class ResolveConstantsWalk {
         ENFORCE(block->body);
 
         auto blockLoc = core::Loc(todo.file, block->body.loc());
-        auto *id = ast::cast_tree<ast::ConstantLit>(block->body);
+        core::ClassOrModuleRef symbol = core::Symbols::StubModule();
 
-        if (id == nullptr || !id->symbol.exists() || !id->symbol.isClassOrModule()) {
+        if(auto *constant = ast::cast_tree<ast::ConstantLit>(block->body)) {
+            if(constant != nullptr && constant->symbol.exists() && constant->symbol.isClassOrModule()) {
+                symbol = constant->symbol.asClassOrModuleRef();
+            }
+        } else if(ast::MK::isTClassOf(block->body)) {
+            send = ast::cast_tree<ast::Send>(block->body);
+
+            ENFORCE(send);
+
+            if(send->numPosArgs() == 1) {
+                auto arg = send->getPosArg(0).deepCopy();
+                if(auto *argClass = ast::cast_tree<ast::ConstantLit>(arg)) {
+                    if(argClass != nullptr && argClass->symbol.exists() && argClass->symbol.isClassOrModule()) {
```

Comment:
```suggestion
                    if (argClass->symbol.exists() && argClass->symbol.isClassOrModule()) {
```

(Same argument for non-`nullptr`-ness as above.)